README.md

```diff
@@ -41,26 +41,24 @@ Underneath the hood it uses several probabilistic graphical modeling and deep le
 techniques. To enable a variety of data storage structures, we employ unique
 hierarchical generative modeling and recursive sampling techniques.
 
-| Important Links                               |                                                                      |
-| --------------------------------------------- | -------------------------------------------------------------------- |
-| :computer: **[Website]**                      | Check out the SDV Website for more information about the project.    |
-| :orange_book: **[SDV Blog]**                  | Regular publshing of useful content about Synthetic Data Generation. |
-| :book: **[Documentation]**                    | Quickstarts, User and Development Guides, and API Reference.         |
-| :octocat: **[Repository]**                    | The link to the Github Repository of this library.                   |
-| :keyboard: **[Development Status]**           | This software is in its Pre-Alpha stage.                             |
-| [![][Slack Logo] **Community**][Community]    | Join our Slack Workspace for announcements and discussions.          |
-| [![][MyBinder Logo] **Tutorials**][Tutorials] | Run the SDV Tutorials in a Binder environment.                       |
+| Important Links                               |                                                                                                     |
+| --------------------------------------------- | ----------------------------------------------------------------------------------------------------|
+| :computer: **[Website]**                      | Check out the SDV Website for more information about the project.                                   |
+| :book: **[Documentation]**                    | Learn how to use the SDV library with tutorials, demos and API references.                          |
```

Comment:
@pvk-developer I wonder why this is wrapping around to a new line right now? Is there any way that the word "Documentation" can be right next to the book icon on the same line?

<img width="223" alt="image" src="https://user-images.githubusercontent.com/3227804/224418851-c83f034c-3d59-48a8-a78d-6c8a51652d6c.png">

Reply:
If we can't get it to fit, I'd suggest we:
- Shorten "Documentation" to "Docs"
- Shorten "DataCebo Blog" to "Blog"

Reply:
@npatki it's because of the length of the text on the description. Here is how it looks after the changes:
![Screenshot from 2023-03-13 13-20-50](https://user-images.githubusercontent.com/41479552/224700600-2d58ba7e-aeef-47b2-84db-f7c559b9ba3d.png)
As you can see now the `Important Links` get wrapped. However, if you open the `README.md` only it shows as this: (ps this is the old version with `datacebo blog` and `documentation`):
![Screenshot from 2023-03-13 13-23-29](https://user-images.githubusercontent.com/41479552/224701017-9288ef24-eb5a-4519-93e7-b96a462e15de.png)

The reason is because of github styling. I did some experiments trying to style it with `html` but still getting similar results. I don't really want to hard code the style as it may look different on different devices as well.

Reply:
Interesting. Let's keep the top version. Even if `REAMDE.md` shows the full version, I think most people will read it on the main branch.